### ISSUE TITLE
Is "Retrieving Tile Information at Runtime with Tiled2Unity" out of date?

### ISSUE THREAD

**Issue by anchpop** [Seanba/Tiled2Unity#115]

**Comment by anchpop** [comment]
Whoops, accidentally hit enter. I meant to say that I can't seem to find `fe-rules.tmx` anywhere, and I can't find information on how to use object layers with automapping rules (a feature that you seem to have added yourself :p) 

Thanks for the wonderful tool, by the way!

**Comment by Seanba** [comment]
Hi Anchpop, the `fe-rules.tmx` file is in the [tiled/FileEmblem](https://github.com/Seanba/Tiled2Unity-TileInfoExample/tree/master/tiled/FireEmblem) folder.

Using object layers is a pretty cool feature of the Tiled map editor (for the record, I didn't add it to Tiled but I think I fixed a bug with it a while back). Once you view the fe-rules.tmx file it should make sense how I went about it.

And yes, that project had gotten old and dusty. I just ran it against Unity 2017 and the latest version of Tiled2Unity and updated some scripts. It should be fresh again on Github. (It should work for you even if you're not on Unity 2017 yet).

Thanks for reminding me to keep these things up to date.
